normalize_repos/labels.yml

```
@@ -0,0 +1,133 @@
+groups:
+  group:
+    name: "priority"
+    labels:
+      label:
+        name: "critical"
+        color: "UNFAVOURABLE"
+        description: "Must be fixed ASAP"
+        emoji: "\uD83D\uDFE5"
+      label:
+        name: "high"
+        color: "NEGATIVE"
+        description: "Stalls work on the project or its dependents"
+        emoji: "\uD83D\uDFE7"
+      label:
+        name: "medium"
+        color: "NEUTRAL"
+        description: "Not blocking but should be fixed soon"
+        emoji: "\uD83D\uDFE8"
+      label:
+        name: "low"
+        color: "POSITIVE"
+        description: "Low priority and doesn't need to be rushed"
+        emoji: "\uD83D\uDFE9"
+  group:
+    name: "status"
+    labels:
+      label:
+        name: "ready for work"
+        color: "LIGHT"
+        description: "Ready for work"
+        emoji: "\uD83C\uDFC1"
+      label:
+        name: "blocked"
+        color: "MEDIUM"
+        description: "Blocked & therefore, not ready for work"
+        emoji: "\uD83D\uDEA7"
```

Please replace emojis that are present in the cheat sheet with the colon-representation. Only leave those that are absent as codepoints.
```suggestion
        emoji: ":construction:"
```